automated install/basic-install.sh

```diff
@@ -81,19 +81,15 @@ PI_HOLE_INSTALL_DIR="/opt/pihole"
 PI_HOLE_CONFIG_DIR="/etc/pihole"
 PI_HOLE_BIN_DIR="/usr/local/bin"
 PI_HOLE_V6_CONFIG="${PI_HOLE_CONFIG_DIR}/pihole.toml"
-if [ -z "$useUpdateVars" ]; then
-    useUpdateVars=false
-fi
+fresh_install=true
```

Comment:
I wonder if this should only be set if it's empty. We have users who run the installer in some kind of automatic setup using the `--unattended` flag and passing a config file (`setupVars.conf`) or now `pihole.toml`.

See here: https://github.com/pi-hole/pi-hole/issues/5060


Maybe this is not necessary, I did not deeply review nor test, but it just came to my mind.

Reply:
One of the paths I tested was running the installer with `--unattended` set and a preconfigured `/etc/pihole/pihole.toml`. Seemed to work OK!

Reply:
Perfect!